src/promptflow/tests/sdk_cli_azure_test/unittests/test_azure_cli_activity_name.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import sys
+import pytest
+
+from promptflow._cli._pf_azure._flow import add_parser_flow
+from promptflow._cli._pf_azure._run import add_parser_run
+from promptflow._cli._utils import _get_cli_activity_name
+
+
+def get_cli_activity_name(cmd):
+    sys.argv = list(cmd)[1:]
```

Comment:
maybe we test on the real parser used by our cli?

Reply:
done